### PR TITLE
Attempted fix for #439

### PR DESCRIPTION
**Pull request by NotExpectedYet**
I couldn't reproduce this, but I've removed the pointless code and added checks in to make sure the tools exists before applying any value. 

I tried 
0: Fresh printer offline with no spool
1: Printer online and printering. 

0: Existing printer offline with no spool
1: Printer online and printering 

0: Existing printer offline with spool attached
1: printer online and printering

0: Fresh printer offline with spool attached
1: printer online and printering. 

Would like confirmation this is fixed from the user.

### CODE REVIEW

File: client_src/js/lib/modules/viewUpdater.js [hunk 1]
```diff
@@ -1000,17 +1000,15 @@ async function updateState(printer, clientSettings, view) {
...
           tool.innerHTML = `${printer.selectedFilament[i].spools.material}`;
         }
       } else {
-        tool.innerHTML = "No Spool";
+        if (tool) {
+          tool.innerHTML = "No Spool";
+        }
       }
     }
-  } else {
```
Comment (NotExpectedYet) [inline client_src/js/lib/modules/viewUpdater.js:1012]: This shouldn't even run so it's pointless. There's no tool boxes filled if it doesn't know the actual tool counts from the printer.

File: client_src/js/lib/modules/viewUpdater.js [hunk 2]
```diff
@@ -1000,17 +1000,15 @@ async function updateState(printer, clientSettings, view) {
       const tool = document.getElementById(`${printer._id}-spool-${i}`);
       if (printer.selectedFilament[i] !== null) {
         const filamentManager = await checkFilamentManager();
-        if (filamentManager) {
```
Comment (NotExpectedYet) [inline client_src/js/lib/modules/viewUpdater.js:1003]: Removed that pointless if function. Surely that was stoner laziness I'd be embarrassed if not XD

File: client_src/js/lib/modules/viewUpdater.js [hunk 3]
```diff
@@ -1000,17 +1000,15 @@ async function updateState(printer, clientSettings, view) {
...
-          tool.innerHTML = `${printer.selectedFilament[i].spools.material}`;
-        } else {
+        if (tool) {
           tool.innerHTML = `${printer.selectedFilament[i].spools.material}`;
         }
       } else {
-        tool.innerHTML = "No Spool";
+        if (tool) {
```
Comment (davidzwa) [inline client_src/js/lib/modules/viewUpdater.js:1007]: `else if` :P

Comment (NotExpectedYet) [reply client_src/js/lib/modules/viewUpdater.js:1007]: Nah can't be. The if is to protect the tool been updating the else is for selected filament been null

Comment (davidzwa) [reply client_src/js/lib/modules/viewUpdater.js:1007]: YOU SHALL NOT! be ... incorrect

**Review by davidzwa — APPROVED** [review]
No repro possible. Looks sorta ok to me

**Pull request merged**